### ISSUE TITLE
When is new version going to be available?

### ISSUE THREAD

**Issue by ronosa** [fyookball/electrum#27]
I just want to get my BCH but can't, and I'm sure there are hundreds more that can't either. Unfortunately I don't know how to run the updates through here. issue#6 is preventing me from sending coins.

**Comment by fyookball** [comment]
in a few days.

**Comment by ronosa** [comment]
Is there any easy way to apply the fix from #6? Or learn how to do it? I need to get these coins out

**Comment by fyookball** [comment]
well you will need to run from source, not a binary file.   do you have git command line installed?

**Comment by ronosa** [comment]
Yes I installed that.

**Comment by fyookball** [comment]
git clone http://github.com/fyookball/electrum.git 

it should create an electrum folder

**Comment by fyookball** [comment]
if you type python from the command prompt, does it run a python console or does it say python not found

**Comment by ronosa** [comment]
python not found on my laptop. have it on my pc. But didn't know what to do after that so didn't bother on laptop.

**Comment by fyookball** [comment]
well if you want to run it before we publish a binary you need python :)

**Comment by ronosa** [comment]
I will get it. But what do I need to do after I have it?

**Comment by fyookball** [comment]
after that you need to cd (change directory) to the electrum folder you just downloaded, and then 

python setup.py install

and then try running:

python electron-cash

**Comment by ronosa** [comment]
Okay I will try. Thanks so much for the help so far.

**Comment by staffnsnake** [comment]
Hi Fyookball,

I downloaded git and you clone. 
I typed cd electrum

Then I wrote python setup.py install and got back 

Traceback (most recent call last):
File "setup-py", line 5, in <module>
from setuptools import setup
ImportError: No module named setuptools

I tried to run it anyway as python electron-cash and got back
Error: No module named dns. Try 'sudo pip install <module-name>'

I didn't know what the module name was that I needed to pip install

**Comment by staffnsnake** [comment]
When I tried to install python anyway, it said I already had version 3.5

**Comment by staffnsnake** [comment]
And may I also use a Mycelium PIP39 seed for this, or do I need to move funds into another computer with electrum, create a new wallet, move funds into that new wallet on electrum, then use the seed from the older now empty wallet? Because every time I have tried that, it shows all those transactions and a zero balance. The same happened using my PIP39 seed from Blockchain.info

**Comment by staffnsnake** [comment]
I managed to install pip.

When I enter python setup.py install it said I didn't have write permission to the directory /usr/local/lib/python2.7/dist-packages/

Do need to sign in as root?

**Comment by staffnsnake** [comment]
Then I tried using the PIP39 seed from the (now empty) Blockchain.info wallet. The wallet loaded fine in electron-cash, showing all the transactions in that wallet leading to its eventual emptying into mycelium. Now it shows four transactions that are not verified and the last transaction being the one emptying the wallet, which is "Unconfirmed". So the balance says what is was before I emptied it, but with the final transaction showing 0BCC if that one confirms eventually. Right now it's hung on the unconfimed/not verified status.

I am starting to think I need to send it all back into electrum on a different computer, then repeat the previous actions of create new wallet, move BTC into that, then use seed of old empty wallet in eletron. Do the old and new BTC wallets for this operation have to be I the same app?
